cdap-docs/developers-manual/source/building-blocks/plugins.rst

```diff
@@ -288,14 +293,16 @@ where ``config.json`` contains:
 
 Note that when deploying a user artifact that extends a system artifact,
 you must prefix the parent artifact name with ``'system:'``.
-This is in case there is a user artifact with the same name as the system artifact.
+This is in the event there is a user artifact with the same name as the system artifact.
 If you are extending a user artifact, no prefix is required.
 
-You can deploy third-party JARs in the same way except the plugin information needs to be explicitly listed.
-As described in the documentation on :ref:`artifacts`, only snapshot artifacts can be
-re-deployed without requiring that they first be deleted.
+You can deploy third-party JARs in the same way except the plugin information needs
+:ref:`to be explicitly listed <plugins-third-party>`. As described in the documentation on
+:ref:`artifacts`, only snapshot artifacts can be re-deployed without requiring that they
+first be deleted.
 
-Using the RESTful API:
+Using the RESTful API (note that the version needs to be set explicitly, as the contents of the
```

Comment:
in a vacuum, this sounds like you always need to send the version.  Maybe qualify it? "note that if the version is not in the jar's manifest, the version needs to be set explicitly..."  Makes it sound very wordy though.